pyccel/ast/cuda.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+CUDA Module
+This module provides a collection of classes and utilities for CUDA programming.
+"""
+from pyccel.ast.core import FunctionCall
+
+__all__ = (
+    'KernelCall',
+)
+
+class KernelCall(FunctionCall):
+    """
+    Represents a kernel function call in the code.
+    
+     The class serves as a representation of a kernel
+     function call within the codebase.
+
+    Parameters
+    ----------
+    func : FunctionDef
+        The definition of the function being called.
+
+    args : tuple
+        The arguments being passed to the function.
+
+    numBlocks : NativeInteger
+        The number of blocks.
+
+    tpblock : NativeInteger
```

Comment:
Should we stick to camelCase for all variables?

Reply:
When I read the source code again, I realized that we work with snake case a lot. So, I changed the variable names to snake case for now 26cf144